OsmAnd-java/src/main/java/net/osmand/router/ExitInfo.java

```diff
@@ -15,11 +16,19 @@ public void setRef(String ref) {
         this.ref = ref;
     }
 
-    public String getExitStreetName() {
-        return exitStreetName;
+    public String getDestinationRef() {
```

Comment:
destination ref has nothing to do with exit info, so it shouldn't be stored here

Reply:
Ok